doc/src/Build_development.rst

```diff
@@ -273,21 +275,22 @@ Unit tests for force styles
 A large part of LAMMPS are different "styles" for computing non-bonded
```

Comment:
@megmcca here is an update of the force style test docs. Please let me know, if there could be additional improvements.